### ISSUE TITLE
Download messages from "Done" tag

### ISSUE THREAD

**Issue by ngonik** [eisenjulian/fb-page-chat-download#3]
Hi! We store all of our chats under the Done tag in Facebook. When I use the script it only downloads the messages in the Inbox and it works perfectly.

Is there any way to download every message in every tag?

Thanks!

**Comment by eisenjulian** [comment]
Hi @ngonik,
According to the [docs](https://developers.facebook.com/docs/graph-api/reference/page/conversations/), they've added a `folder` and `tags` parameters that can be used to do what you want. Can you modify the target URL at [line 28](https://github.com/eisenjulian/fb-page-chat-download/blob/master/run.py#L28) to add those parameters and test it out? I'm not sure in which it should be added, but it should be easy to try it out. If it works we can add it as params to the script, PRs are of course welcome.

**Comment by mds-** [comment]
Hi @eisenjulian @ngonik ,
Great script, thanks. I ran into the same problem...I tried as eisenjulian suggested to change the URL and it appears to be working with the "Done" messages with the addition of '&folder=page_done' as below. Sorry I'm not familar enough with python and git to do a PR at the moment.

 self.uri = self.build_url('{}/conversations?fields=participants,link&limit=400&folder=page_done', page)
